bcipy/helpers/stimuli.py

```diff
@@ -174,13 +187,15 @@ def best_case_rsvp_inq_gen(alp: list,
         # append colors
         colors.append([color[i] for i in range(len(color) - 1)] +
                       [color[-1]] * stim_length)
-    return (samples, times, colors)
+    return InquirySchedule(samples, times, colors)
 
 
-def random_rsvp_calibration_inq_gen(alp, timing=[0.5, 1, 0.2],
+def random_rsvp_calibration_inq_gen(alp,
```

Comment:
What do you think of moving these into `task/rsvp` with the decisionmaker etc... It doesn't need to happen in this PR, but thinking of where best to put  them. I'm thinking `helpers/` module should only hold core items or connections between modules, these are very task-specific.

Reply:
I like that suggestion